src/Features/Core/Portable/Completion/CompletionServiceWithProviders.cs

```diff
@@ -194,7 +194,15 @@ internal protected CompletionProvider GetProvider(CompletionItem item)
             {
                 lock (_gate)
                 {
-                    _nameToProvider.TryGetValue(name, out provider);
+                    if (!_nameToProvider.TryGetValue(name, out provider))
+                    {
+                        // Sometimes the _nameToProvider map hasn't been fully initialized in
+                        // debugger scenarios. If we couldn't find the provider, ensure everything
```

Comment:
hrmm.. that makes me feel like that's an issue in that code path.  How can the debugger sidestep the initialization of the providers?